### PR TITLE
fix for gallery info panel covering dialog/menu controls in narrow viewport (RSDEV-1088)

### PR DESCRIPTION
**Pull request by mKowalski256**
### Problem
The fix for [RSDEV-1088](https://researchspace.atlassian.net/browse/RSDEV-1088).

On mobile portrait viewports the floating info panel blocked controls beneath it: the **Move** dialog buttons were unclickable, and the last items on Actions menu, like the **Delete** item, were hidden. This is reproduceable in desktop view, if you narrow the UI below 600px.

**Cause:** the panel's `SwipeableDrawer` hard-coded `zIndex: 1400`, above MUI's dialog/menu layer (`1300`), so its closed peek strip floated over the bottom of any open dialog or menu. The `1400` was only needed so the panel floats above the **picker dialog**.

### Solution
- `InfoPanel.tsx`: removed the explicit z-index so the drawer uses the default (`theme.zIndex.drawer`), below dialogs/menus on the main page.
- `picker/index.tsx`: wrap `MainPanel` in a `ThemeProvider` that raises the band only for the picker (drawer just above the dialog, modal just above the drawer), so the panel floats above the picker dialog while its own menus/dialogs stay above the panel.

Result: panel sits below action overlays in both contexts. `tsc` + lint clean, existing ActionsMenu/MoveDialog tests pass.

### CODE REVIEW

File: src/main/webapp/ui/src/eln/gallery/picker/index.tsx [hunk 1]
```diff
@@ -183,24 +193,26 @@ const Picker = observer(
...
-                        orderBy={orderBy}
-                        setSortOrder={setSortOrder}
-                        setOrderBy={setOrderBy}
-                        appliedSearchTerm={appliedSearchTerm}
-                        setAppliedSearchTerm={setAppliedSearchTerm}
-                      />
+                      <ThemeProvider theme={raisedZIndexTheme}>
+                        <MainPanel
+                          selectedSection={selectedSection}
+                          path={path}
```
[the remark this answers is not in the document]
Comment (mKowalski256) [reply src/main/webapp/ui/src/eln/gallery/picker/index.tsx]: Good catch, addressed. To be precise this was pre-existing rather than introduced here: before this PR the info panel was hard-coded to z-index 1400 in the picker, so the sidebar's 1300 menus/dialogs already sat below it. The PR moved the panel to 1301, keeping the same ordering relative to the sidebar.

Still worth closing for consistency, so I moved the raised-zIndex `ThemeProvider` up to wrap both `Sidebar` and `MainPanel`. The sidebar's menus and dialogs now use the raised modal z-index (1302), above the info panel (1301). The sidebar drawer keeps its own hard-coded z-index 1300, so only its overlays are raised.

File: src/main/webapp/ui/src/eln/gallery/picker/index.tsx [hunk 2]
```diff
@@ -47,6 +47,16 @@ const Picker = observer(
     const sidebarId = React.useId();
     const viewport = useViewportDimensions();
     const selection = useGallerySelection();
+    const theme = useTheme();
+    // Float the info panel above this dialog, but keep its menus/dialogs above
+    // the panel: drawer just over the dialog, modal just over the drawer.
+    const raisedZIndexTheme = React.useMemo(
+      () => ({
+        ...theme,
+        zIndex: { ...theme.zIndex, drawer: theme.zIndex.modal + 1, modal: theme.zIndex.modal + 2 },
```
[the remark this answers is not in the document]
Comment (mKowalski256) [reply src/main/webapp/ui/src/eln/gallery/picker/index.tsx:56]: This is pre-existing rather than introduced here, so out of scope for this PR. Before this change the info panel was hard-coded to z-index 1400 in the picker, so it already sat above the sidebar drawer's 1300; this PR moves it to 1301, keeping the same ordering relative to the sidebar (and on the main page it now drops below the sidebar, which is an improvement). The case is also narrow: small viewport, a file selected so the 80px peek is showing, and the nav drawer open, where the peek mostly overlaps empty space below the section list. Making the open nav drawer sit above the peek would mean giving the shared Sidebar drawer a stateful, higher-than-panel z-index, which is disproportionate here. Leaving it as a known pre-existing edge case.

**Comment by mKowalski256** [comment]
Test instance running at https://rsdev-1088-mobile-gallery-6159af84-5.researchspace.com, the action menu and 'move' dialog now appears above info panel, so buttons on the bottom part are not hidden.

**Review by tyteen4a03 — APPROVED** [review]
Not a huge fan of the house of cards but I'll approve it for now.

**Review by tchoi-rs — APPROVED** [review]
